lemur/plugins/lemur_vault_dest/plugin.py

```diff
@@ -22,6 +22,8 @@
 from cryptography import x509
 from cryptography.hazmat.backends import default_backend
 
+from validators.url import regex as url_regex
```

Comment:
nitpick: if I understood the import logic correctly, grouping third parties libraries would be better here than creating a group for each one